samples/snippets/snippets.py

```diff
@@ -230,6 +230,160 @@ def count_query_with_stale_read(client):
     return [task1, task2, task3]
 
 
+def sum_query_on_kind(client):
+    # [START datastore_sum_aggregation_query_on_kind]
+    # Set up sample entities
+    task1 = datastore.Entity(client.key("Task", 4871162878230528))
+    task2 = datastore.Entity(client.key("Task", 5069834475798528))
+    task3 = datastore.Entity(client.key("Task", 5081353846521856))
+
+    task1["hours"] = 5
+    task2["hours"] = 3
+    task3["hours"] = 1
+
+    tasks = [task1, task2, task3]
+    client.put_multi(tasks)
```

Comment:
Do datastore samples typically include this set-up part?

Reply:
Yes, as part of making samples [copy-paste-runnable](https://googlecloudplatform.github.io/samples-style-guide/#introduction). This matches the style of the Java and Python [count() samples](https://cloud.google.com/datastore/docs/aggregation-queries#count_aggregation), for example.